src/lab2/task1/Complex.java

```diff
@@ -0,0 +1,42 @@
+package lab2.task1;
+
+public class Complex {
+    private int real;
+    private int imaginary;
+
+    public int getReal() {
+        return real;
+    }
+
+    public void setReal(int real) {
+        this.real = real;
+    }
+
+    public int getImaginary() {
+        return imaginary;
+    }
+
+    public void setImaginary(int imaginary) {
+        this.imaginary = imaginary;
+    }
+
+    public Complex(int real_nr, int imaginary_nr) {
```

Comment:
in Java trebuie sa folosesti camelCase, nu snake_case. Totodata, astea ar trebui numite tot real si imaginary, deci ar fi:
```
public Complex(int real, int imaginary) {
  this.real = real;
  this.imaginary = imaginary;
}
```
tocmai de aia folosim `this` ca sa dezambiguizam variabilele cu acelasi nume